src/gui/addnewtorrentdialog.cpp

```diff
@@ -976,20 +976,23 @@ void AddNewTorrentDialog::setupTreeview()
             m_ui->contentTreeView->setExpanded(currentIndex, true);
         }
 
-        // Check file name blacklist for torrents that are manually added
-        QVector<BitTorrent::DownloadPriority> priorities = m_contentModel->model()->getFilePriorities();
-        Q_ASSERT(priorities.size() == m_torrentInfo.filesCount());
-
-        for (int i = 0; i < priorities.size(); ++i)
+        if (BitTorrent::Session::instance()->isExcludedFileNamesEnabled())
```

Comment:
Diff in this file is confusing (to me at least). I just wrapped this section in this.